tests/test_validation_tool.py

```diff
@@ -191,9 +191,7 @@ def test_cli_abort_on_error() -> None:
         ]
     )
     assert isinstance(ret, str)
-    assert re.match(
-        r"^(tests/data/ethernet/frame/invalid/).+(\.raw) (classified as FalseNegative)$", ret
-    )
+    assert re.match(r"^(1 messages were classified incorrectly)$", ret)
```

Comment:
I changed it back in 60bb6d8 so that the path is printed and an indication that the test was aborted. But since this test is supposed to test the abort_on_error flag, every other value than one in this case would have been wrong.